_posts/2017-02-06-the-4-key-elements-of-a-conversation.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+asset-type: post
+name: The 4 key elements of a conversation
+title: 'The 4 key elements of a conversation'
+date: 2017-02-08 09:00:00 +00:00
+author: Giulia Mantuano
+image:
+  src: /assets/img/custom/blog/2017-02-06-conversational-interface-design-fundamentals.jpg
+tags:
+- UX
+- Chatbot
+- Conversation Design
+- Conversational Interfaces
+
+---
+Conversational Interfaces is a term often used to define systems that hold "human-like" conversations, examples include; Chatbots, Voice User Interfaces, Embodied Conversational Agents. The user interacts with these using text input, voice recognition, and gestures. These types of interfaces are often unable to provide a good conversational experience. In order to create a better experience we need to be aware of the following 4 key elements of human-to-human conversations.
+
+## 1. Turn-taking
+> A turn is one interaction between the user and the system, and a conversation is made of at least two-turns.
+
+<div class="image-side-by-side-text">
+  <img src="/assets/img/custom/blog/2017-02-06-conversational-interface-design-fundamentals/poncho-chatbot.jpg" alt="Hi Poncho on Facebook Messenger" width="250" height="300" />
```

Comment:
Hi Giulia, since we merged yesterday the baseurls PR, now we have to specify {{site.baseurl}} for the relative links. This tag needs to change to: 

```<img src="{{site.baseurl}}/assets/ .... lblablablabla... />"```

That applies to all the `img src` in this page.